tests/x11/multi_users_dm.pm

```diff
@@ -75,16 +75,17 @@ sub run {
     # Make sure screen changed before calling handle_login function (for slow workers)
     wait_still_screen;
     handle_login($user, 1);
+    send_key 'esc'                        if match_has_tag('gnome-activities');
     handle_welcome_screen(timeout => 120) if (opensuse_welcome_applicable);
     assert_screen 'generic-desktop', 60;
     # verify correct user is logged in
     x11_start_program('xterm');
-    wait_still_screen;
+    assert_and_click('xterm-white-area');
     enter_cmd "whoami|grep $user > /tmp/whoami.log";
     assert_script_sudo "grep $user /tmp/whoami.log";
     # logout user
     handle_logout;
-    wait_still_screen;
+    wait_still_screen 10;
```

Comment:
Just nipticking, why the extra wait?

Reply:
> Just nipticking, why the extra wait?

this extra wait concerns the runs in arch64 that fail sporadically here

Reply:
I can live with this as a short-term workaround, as 12715 was said to require more work (to not have the gnome-activities tag on the generic-desktop needles)

Reply:
Please mention that in a comment, otherwise someone might "optimize" it away

Reply:
updated in the description

Reply:
@punkioudi can you add it to the comments in the code?